_episodes/05-accelerating-lammps.md

```diff
@@ -401,67 +432,55 @@ Not surprisingly, the syntax we use is similar to that of **USER-OMP** package:
 
```

Comment:
@prithwish-ichec, the command `{{ site.run_openmp }} {{ site.sched_lammps_exec }} -in in.lj -sf gpu -pk gpu 2 neigh yes newton off split 1.0`... what is the correct start to this command? Front end shows `-in in.lj -sf gpu -pk gpu 2 neigh yes newton off split 1.0`. I thinks it is `lmp`, but please confirm run_openmp and lammps_exec commands for this.

Reply:
{{ site.run_openmp }} = srun
{{ site.sched_lammps_exec }} = lmp

Reply:
Issues with abstractions remain. Will stage in next merge.